HPXMLtoOpenStudio/resources/EPvalidator.xml

```diff
@@ -952,6 +952,8 @@
       <sch:assert test='count(h:EnergyFactor) + count(h:CombinedEnergyFactor) &lt;= 1'>Expected 0 or 1 element(s) for xpath: EnergyFactor | CombinedEnergyFactor</sch:assert>
       <sch:assert test='(count(h:EnergyFactor) + count(h:CombinedEnergyFactor) + count(h:ControlType) = 0) or (count(h:EnergyFactor) + count(h:CombinedEnergyFactor) + count(h:ControlType) = 2)'>Expected 0 or 2 element(s) for xpath: EnergyFactor | CombinedEnergyFactor | ControlType</sch:assert>
       <sch:assert test='count(h:extension/h:UsageMultiplier) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/UsageMultiplier</sch:assert>
+      <sch:assert test='count(h:extension/h:IsVented) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/IsVented</sch:assert>
+      <sch:assert test='count(h:extension/h:VentedFlowRate) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/VentedFlowRate</sch:assert>
```

Comment:
These ought to be nested checks (i.e., VentedFlowRate is only of interest if IsVented is true).

Reply:
@shorowit do you have an example of how to do this?

Reply:
You've done this before. I'm just asking for something like:

```
 <sch:pattern name='[ClothesDryer]'>
    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Appliances/h:ClothesDryer'>
      ...
      <sch:assert test='count(h:extension/h:IsVented) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/IsVented</sch:assert> <!-- See [VentedClothesDryer] -->
    </sch:rule>
  </sch:pattern>

  <sch:pattern name='[VentedClothesDryer]'>
    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Appliances/h:ClothesDryer[h:extension/h:IsVented="true"]'>
      <sch:assert test='count(h:extension/h:VentedFlowRate) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/VentedFlowRate</sch:assert>
    </sch:rule>
  </sch:pattern>
```